src/SFA.DAS.LevyTransferMatching.Web/Validators/Opportunities/ApplicationDetailsPostRequestAsyncValidator.cs

```diff
@@ -45,11 +45,13 @@ public ApplicationDetailsPostRequestAsyncValidator(IOpportunitiesService opportu
                         return false;
                     }
 
-                    return result.Opportunity.RemainingAmount >= selectedStandard.ApprenticeshipFunding
-                            .GetEffectiveFundingLine(model.StartDate.Value)
-                            .CalcFundingForDate(model.ParsedNumberOfApprentices, model.StartDate.Value);
+                    return model.StartDate.HasValue
```

Comment:
You _could_ simplify this expression a little by returning `!model.StartDate.HasValue || result.Opp..... `

This is a minor observation though and I'm happy whether or not it is tweaked.

Reply:
Thanks for your feedback Chris.  Your suggestion occupies less lines of code definitely.  I am not sure for some reason, I think that the current solution is more readable?  So I would leave it as is if that is acceptable.  May be it is just me?  I am happy to change it though as you like.  Thanks

Reply:
Testing my nickname - Ignore